src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/button-has-type */
```

Comment:
- [ ] Kindly avoid disabling linters manually, please remove this line as there should be no custom change to the linters 👍🏿